manifests/instance/config.pp

```diff
@@ -163,6 +165,19 @@
     }
   }
 
+  # configure JMX
+
+  $jmxremote_defaults = {
+    'port'         => '9010',
+    'local.only'   => true,
+    'authenticate' => false,
+    'ssl'          => false,
+  }
```

Comment:
In case of multiple instances, won't this fail because each instance will try to listen onto the same port ?